### PR TITLE
i-figured:0.1.0

### PR DESCRIPTION
**Pull request by RubixDev**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Easy and customizable per-section figure numbering. @PgBiel and @aagolovanov are included as authors as the code is based off code they posted on the Discord.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

**Comment by PgBiel** [comment]
Feel free to keep only yourself as author; me and @aagolovanov only provided the original snippet, but you're the one packaging things and providing improvements. :)

It's probably more appropriate to keep a Credits section in README.md instead.

**Comment by RubixDev** [comment]
Okay, I've added an "Acknowledgements" section in the README and removed you as authors.

File: packages/preview/i-figured/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "i-figured"
+version = "0.1.0"
+authors = ["RubixDev"]
+entrypoint = "i-figured.typ"
+keywords = ["figure"]
+license = "MIT"
+repository = "https://github.com/RubixDev/typst-i-figured"
+description = "Configurable figure numbering per section."
```
Comment (laurmaedje) [inline packages/preview/i-figured/0.1.0/typst.toml:9]: same as on the other PR, could you exclude the example files with the `exclude` key?

Comment (RubixDev) [reply packages/preview/i-figured/0.1.0/typst.toml:10]: done (sorry, here it's a force push instead of a simple commit)

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**